pkgs/by-name/li/libstudxml/package.nix

```diff
@@ -0,0 +1,28 @@
+{ lib
+, stdenv
+, fetchgit
+, build2
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "libstudxml";
+  version = "1.1.0-b.10+2";
+
+  src = fetchgit {
+    url = "https://git.codesynthesis.com/libstudxml/libstudxml.git";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-OsjMhQ3u/wLhOay7qg9sQMEhnAOdrO30dsKQ8aDWUOo=";
+  };
+
+  outputs = [ "out" "dev" "doc" ];
+
+  nativeBuildInputs = [ build2 ];
+
+  meta = {
+    description = "A streaming XML pull parser and streaming XML serializer implementation for modern, standard C++";
+    homepage = "https://www.codesynthesis.com/projects/libstudxml/";
+    license = lib.licenses.mit;
+    maintainers = with lib.maintainers; [ tomasajt ];
+    platforms = lib.platforms.all;
```

Comment:
```suggestion
  meta = with lib; {
    description = "A streaming XML pull parser and streaming XML serializer implementation for modern, standard C++";
    homepage = "https://www.codesynthesis.com/projects/libstudxml/";
    license = licenses.mit;
    maintainers = with maintainers; [ tomasajt ];
    platforms = platforms.all;
```

Reply:
I'd rather not, sorry. I know it's widely used, but I like being more explicit. I usually only use `with` right before lists.

Reply:
Fair enough, I'm actually seeing more of this type of thing these days :)